### ISSUE TITLE
Create a functional base test class based on HdxBaseTest

### ISSUE THREAD

**Issue by alexandru-m-g** [OCHA-DAP/hdx-ckan#3852]
This will automatically load all the HDX plugins so that new tests won't need to override the _load_plugins() function. 

This will still allow the flexibility for each test class to only inherit from HdxBaseTest and still only load the plugins it needs. 

**Note** Extensions that we plan to make available for the public should NOT create tests based on this new functional base test class

**Comment by danmihaila** [comment]
@brew this is low priority issue. Please try to finish the other assigned issues before starting this one
